File: app/src/main/java/ma/ofppt/devoamm204/MainActivity.java
package ma.ofppt.devoamm204;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        new Thread(new Runnable() {
            @Override
            public void run() {
               // TODO : BG Task
                //...
                runOnUiThread(new Runnable() {
                    @Override
                    public void run() {

                    }
                });
                //...
            }
        }).start();

      new MyThread().start();
    }
}

File: app/src/main/java/ma/ofppt/devoamm204/MainActivity.kt
package ma.ofppt.devoamm204

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import ma.ofppt.devoamm204.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding
    lateinit var  f1 : F1Fragmenty
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        f1 = F1Fragmenty()
        val i= 0;
       var patient = Patient.getLstPatients().get(i);
        val b = Bundle()
        b.putString("cin",patient.cin)
        b.putString("nom",patient.nom)
        b.putInt("age",patient.age)
        f1.arguments =b
        supportFragmentManager.beginTransaction().replace(R.id.fragments,f1).commit()
    }
}